tests/test_selenium_blog_page.py

```diff
@@ -0,0 +1,59 @@
+from page_objects.home_page import HomePage
+import pytest
+
+
+@pytest.mark.smoke
+def test_open_post(create_driver):
+    """
+    open site -> go to blogs -> open post
+    """
+    driver = create_driver
+    blog_page = HomePage(driver).click_on_blogs()
+    blog_page.click_on_article_aikon()
+    assert blog_page.article_title_display(), 'Not element is displaying'
```

Comment:
never check only the presence of the element. You should check the value. From a test perspective displaying is not proving correct behaviour 
please apply this suggestuion to all tests